src/main/java/org/jsoup/JsoupOptions.java

```diff
@@ -0,0 +1,42 @@
+package org.jsoup;
+
+/**
+ The Jsoup options.
+
+ @author Daniel Kurka */
+public class JsoupOptions {
```

Comment:
Hi Daniel,
              Please correct if I am wrong, why donot we use ENUM instead of class for JsoupOptions

Cheers!!!